boxes/templates/boxes/closed.html

```diff
@@ -7,15 +7,12 @@
       <div class="row center-xs">
         <div class="col-xs-10 col-sm-8 col-lg-7">
           <h3 class="subtitle top text-darkest">This link is no longer active.</h3>
-          <h4 class="text-blue sub">The reason could be due to one of the following possibilities:</h4>
+          <h4 class="text-blue sub">That could be because:</h4>
           <ul class="close-box">
-            <li>The link expired</li>
-            <br>
+            <li>- The link expired</li>
             <li>The recipient closed or deactivated the link.</li>
```

Comment:
I think the `-` isn't needed since it is inside a list. The bullet point stuff should be handled in css. no?